scipy/special/cephes/pdtr.c

```diff
@@ -145,20 +145,18 @@ double m;
 }
 
 
-double pdtr(k, m)
-int k;
-double m;
+double pdtr(double k, double m)
 {
     double v;
 
-    if ((k < 0) || (m < 0.0)) {
+    if ( k < 0.0 || m < 0.0 ) {
```

Comment:
If you want to delete the parens, also get rid of the spaces.

Reply:
done.